docs/api/public_php_api.md

```diff
@@ -1075,6 +1075,48 @@ $contentUpdateStruct->setField( 'body', $newfrenchbody );
 
 Since we don't specify a locale for the last two fields, they are set for the `UpdateStruct`'s `initialLanguageCode`, fre-FR.
 
+#### delete translations
+
+##### delete translations from a Content item version
+
+To delete translations from a Content item version, use the `deleteTranslationFromDraft` method on `ContentService`.
+
+```
+public function deleteTranslationFromDraft(VersionInfo $versionInfo, $languageCode);
```

Comment:
Just got an idea, let's use PHP7+ syntax to document method:
`public function deleteTranslationFromDraft(VersionInfo $versionInfo, string $languageCode) : Content`